data/hai/hai reveal 5 philosophers.txt

```diff
@@ -633,124 +492,96 @@ fleet "Smuggler (Hai Traffickers)"
 	names "civilian"
 	cargo 0
 	personality timid
-	variant 3
+	variant 7
 		"Blackbird (Hai Trafficker)"
-	variant 2
-		"Clipper (Hai Trafficker)"
-	variant 2
+	variant 7
+		"Mule (Hai Trafficker)"
+	variant 7
 		"Freighter (Hai Trafficker)"
-	variant 1
-		# The Corvettes are infrequent because they're nearly impossible to catch and serve mostly to show that the smugglers are desperate to avoid capture.
+	variant 5
+		"Clipper (Hai Trafficker)"
+	variant 5
 		"Corvette (Hai Trafficker)"
+	variant 3
+		"Bounder (Hai Trafficker)"
+	variant 3
+		"Star Queen (Hai Trafficker)"
 
 
-mission "Hai Reveal [B06-long] Boarded Smuggler"
-	name "Learn about Keystones"
-	description "You need to know what the message means by asking Sayari or Teeneep on <planet> before going back to Giti."
+mission "Hai Reveal [B07-1] Boarded Smuggler"
+	name "0.184 steradians"
+	description "Report the information you found to Giti on <planet>."
 	boarding
-	destination Stormhold
+	destination Earth
 	clearance
 	to offer
-		has "Hai Reveal [B06-long] Tarazed Analysis: active"
+		has "Hai Reveal [B06] Lovelace Analysis: active"
 	source
 		government "Smuggler (Hai Trafficker)"
 	on offer
+		log `A doomed Hai in a smuggler's vessel yelled "0.184 steradians" as their last words. That is a typical hyperspace aperture size, so it may be the information Giti needs to find Devil-Run.`
```

Comment:
I thought it was _not_ typical?

Reply:
The reason Captain Last recognizes the solid angle is that it is a typical aperture size. If it was an atypical one, then Captain Last would not make the connection.

Recall that there are hundreds of billions of stars in the milky way galaxy. If you resort to random jumping, you'll never find the few key systems  you want. To find this wormhole system intentionally, someone would already need to know what they're looking for. They'd also need the technology to search for it.

Reply:
Simply the use of a solid angle (which is not used for hardly anything of note) is enough to make a connection.
It *cannot* be a *typical* one, or everyone whose ever flown by with an autopilot system would have found it trivially.

Reply:
@MasterOfGrey I'm not sure what you want. Can you make a suggested change for me to commit?

Reply:
This would quite simply align it with the current other text.
```suggestion
		log `A doomed Hai in a smuggler's vessel yelled "0.184 steradians" as their last words. That could be an unusual hyperspace aperture size, so it may be useful information for Giti to find Devil-Run.`
```